src/redux/books/books.js

```diff
@@ -40,27 +21,39 @@ const booksReducer = (state = initialState, action) => {
 };
 
 export const addBooks = (book) => async (dispatch) => {
-  apis.addBooks(book);
-  dispatch({
-    type: ADD_BOOK,
-    book,
-  });
+  try {
+    await apis.addBooks(book);
+    dispatch({
+      type: ADD_BOOK,
+      book,
+    });
+  } catch (error) {
+    console.log(error);
+  }
```

Comment:
- [ ] Great job working so far but I think you are missing to import the styles here since I couldn't see your design the same as the UI given in the requirements and with different color for the button please fix that to make it exactly looks like the same because it is very important to make your project well styled for user experience(UX) 😊♻🔥💯📌

![image](https://user-images.githubusercontent.com/58553711/215302021-c0073300-d9fd-4675-98be-b15a5a70f0d0.png)

- Your design
### I could see alot of differnces here 

- The alignment for the items is column not row ✅
- The colors for the button is blue ✅
- The navbar it should be space-between and above the books ✅

![image](https://user-images.githubusercontent.com/58553711/215301968-83003889-6327-44f4-94c5-3460948bf214.png)


- Given design
![image](https://user-images.githubusercontent.com/58553711/215301989-01738612-14fa-45a6-816c-bcc61f0506e1.png)

Reply:
Done. Pls Approve.  Thanks a lot. @Reem-lab